src/plugins/intel_gpu/tests/functional/shared_tests_instances/behavior/executable_network/exec_graph_info.cpp

```diff
@@ -1,28 +0,0 @@
-// Copyright (C) 2018-2023 Intel Corporation
-// SPDX-License-Identifier: Apache-2.0
-//
-
-#include <common_test_utils/test_constants.hpp>
-#include "behavior/executable_network/exec_graph_info.hpp"
-
-namespace {
-
-using namespace ExecutionGraphTests;
-
-INSTANTIATE_TEST_SUITE_P(smoke_serialization, ExecGraphSerializationTest,
```

Comment:
@olpipi is there an equivalent for API2.0? It should serialize runtime_model

Reply:
As I can see, API 2.0 does not allow to serialize runtime_model. Please let me know, if it is possible

Reply:
@olpipi runtime model is a simple ov::Model which has `serialize` API